src/v3/src/components/WebAuthNSubmitButton/WebAuthNSubmitButton.tsx

```diff
@@ -24,20 +24,24 @@ import {
   UISchemaElementComponent,
   WebAuthNButtonElement,
 } from '../../types';
+import { getTranslation } from '../../util';
 
 const WebAuthNSubmit: UISchemaElementComponent<{
   uischema: WebAuthNButtonElement
 }> = ({ uischema }) => {
+  const { translations = [] } = uischema;
   const { options } = uischema;
 
+  const btnLabel = getTranslation(translations);
+  const btnRetryLabel = getTranslation(translations, 'retry-label');
+
   const { setMessage } = useWidgetContext();
   const onSubmitHandler = useOnSubmit();
   const [waiting, setWaiting] = useState<boolean>(false);
+  const [label, setLabel] = useState<string>(btnLabel!);
 
   const executeNextStep = () => {
-    if (options?.showLoadingIndicator) {
```

Comment:
Is this option just no longer needed?

Reply:
Exactly, it is no longer needed, originally this was added because this component/element was going to be re-used but it ended up being too specific, because of this, it is now useless.